servicekiller.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"time"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/client-go/kubernetes"
+
+	"github.com/spotahome/kooper/log"
+	"github.com/spotahome/kooper/operator/controller/leaderelection"
+)
+
+const (
+	leaderElectionKey        = "knetblackmon-servicekiller-leader"
+	resyncIntervalSecondsDef = 30
+	periodSeconds            = 60
+)
+
+// PodKiller will kill pods at regular intervals.
+type ServiceKiller struct {
+	k8sCli    kubernetes.Interface
+	logger    log.Logger
+	Service   *corev1.Service
+	Namespace string
+}
+
+func NewServiceKiller(kubeCli kubernetes.Interface, namespace string, srv *corev1.Service, logger log.Logger) *ServiceKiller {
```

Comment:
Si la struct est publique et que tu ne fais aucune action d'init, le constructeur n'apporte rien.

Reply:
maybe it's... for the moment :)
Or do we put it in private?